### ISSUE TITLE
Desktop client (Linux and Windows tested) does not resume sync after low disk space is solved.

### ISSUE THREAD

**Issue by sadr0b0t** [nextcloud/desktop#512]
Hello,

I have Ubuntu Linux 16.04 running Next Cloud desktop client 2.3.3.

Once I get low disk space (about 300Mb left) while Next Cloud sync is not yet finished, the Next Cloud stops to sync. This is ok according to docs:
https://docs.nextcloud.com/desktop/2.2/advancedusage.html#low-disk-space

What is not ok:
- Next cloud client does not show error, but it shows that sync is finished fine.
- Settings > Events > Not synced tab does not show skipped files (the list is empty).
- Client does not continue sync after low disk space problem is resolved, even if I stop/restart sync process or select "force sync" option in the settings dialog.
- I can only fix this issue by switching off/on non-synced catalog in the sync list and redownloading the whole catalog again.

Problematic dir contains temp files:
> /home/benderamp/Documents/nextcloud/Автозагрузка/Camera/.VID_20180726_195551.mp4.~53167cb8
> /home/benderamp/Documents/nextcloud/Автозагрузка/Camera/.VID_20180726_202304.mp4.~133be06c
> /home/benderamp/Documents/nextcloud/Автозагрузка/Camera/.VID_20180726_202304.mp4.~409f1526
![2018-07-26 22-54-07](https://user-images.githubusercontent.com/1653222/43285918-2637dcf8-9129-11e8-8c81-fef6c08412e8.png)

**Comment by BierGitte** [comment]
I can confirm this behaviour for Windows client 2.3.2 (1).
I have also not found a way to resume sync, even restart didn't help.

**Comment by sadr0b0t** [comment]
> Settings > Events > Not synced tab does not show skipped files (the list is empty).

I have noted that actually some skipped files appear in Not synced tab, but they go away from it very fast without problem being actually fixed.

**Comment by Skywalker-11** [comment]
Same issue with Windows 10 client version 2.3.3(1)
After account ran out of assigned space and then increasing it the client still shows sync problems due to insufficient storage space:
`12.11.2018 13:29:17,                              xxxx,                     Nextcloud,Das Element ist aufgrund vorheriger Fehler nicht synchronisiert: Error transferring https://cloud.example.com/remote.php/dav/files/userx/xxxxx - server replied: Insufficient Storage (Insufficient space in /xxxxx, 16775 required, 1780 available)`

The user now has >6GB free storage assigned to the account and server has >100GB free so that shouldn't be a problem.
The force synchronization scans the local files and then writes again the above error for previously not sync files.
Only renaming of folders/files helps to sync them again.

**Comment by chainria** [comment]
This problem is still evident. I mean, really? Come on guys. All it takes to completely and irreversibly break syncing is one single time to run out of space. Now what? Start over again? Not really an option, when you don't even know what might have changed on your local machine!

Is there really NO way for the client to redownload the list of files from the server? Because apparently, there isn't. That's really bad for a program which I have trusted my files to.

**Comment by agross** [comment]
This happened today. The process went as far as I remember:

1. I copied some files unrelated to NC filling up my disk (one could just `dd if=/dev/zero` to the same filesystem)
2. The sync client showed notifications that directories larger than the 500 MB I have set showed up (What?)
3. When I opened the client some (i.e. large) of my synced directories became unchecked and showed up in red
4. I needed to restore the checkmarks in front of the respective directories and clicked Apply
5. The sync client tried to redownload all the files in the directories above, actually **doubling the amount of disk space** taken up by my NC sync directories. It did that by creating a `.<file>.<ext>.~<something>` file for each `<file>.<ext>`.

This is a serious bug, IMHO. What is the reasoning behind filling the disk even more when there is already little space left? And why is it required to redownload all the files that have been perfectly in sync seconds before the disk got full?
